props/prop_base.h

```diff
@@ -74,7 +74,7 @@ class PropBase : CommandParser, Looper, protected SaberBase, public ModeInterfac
     current_mode = this;
 #ifdef MENU_SPEC_TEMPLATE
     MKSPEC<MENU_SPEC_TEMPLATE>::SoundLibrary::init();
-#endif    
+#endif
```

Comment:
Sorry all these whitespace corrections showed up. I just overpasted the whole file.